mne/viz/evoked.py

```diff
@@ -1875,6 +1875,9 @@ def plot_compare_evokeds(evokeds, picks=None, gfp=False, colors=None,
     _validate_type(vlines, (list, tuple), "vlines", "list or tuple")
 
     picks = [] if picks is None else picks
+    picked_a_type = isinstance(picks, str) and picks in _DATA_CH_TYPES_SPLIT
+    if title is None and picked_a_type:
```

Comment:
Why is the type check necessary? It's only gonna be in _DATA_CH_TYPES_SPLIT if it's a str anyways, and if it's not a str, it'll simply return false right?

And then you can drop 1878 and just have:

`if title is None and picks in _DATA_CH_TYPES_SPLIT:`